clients/python/src/kaskada/materialization.py

```diff
@@ -94,6 +98,10 @@ def __init__(
             namespace (str): pulsar namespace. defaults to "default".
             topic_name (str): final part of topic url. defaults to a randomly generated uuid
             broker_service_url (str): url to connect to pulsar broker. defaults to "pulsar://127.0.0.1:6650"
+            admin_service_url (str): the pulsar admin REST URL for the cluster. defaults to http://127.0.0.1:8080
+            auth_plugin (str): authentication plugin to use. e.g. "org.apache.pulsar.client.impl.auth.AuthenticationToken"
+            auth_params (str): authentication parameters. e.g. "token:xxx"
+            certificate_chain (str): a custom certificate chain to authenticate the server in TLS connections.
```

Comment:
Wren already maps these to sparrow correctly?

Reply:
wren passes destinations through to sparrow as-is.  doesn't do any mapping.